scripts/buildings/building.gd

```diff
@@ -28,18 +33,21 @@ func _ready() -> void:
 	_generate_states()
 	if building_data == null:
 		state_machine.change_state("Empty")
-	elif is_prebuilt:
```

Comment:
Is there no edge case for just plopping down a building outside of the queue in a pre-built state?

Reply:
Scaffolding probably, we should make it a thing, but will do a story for this